.github/workflows/build-release-alpine-nodejs.yml

```diff
@@ -61,28 +81,17 @@ jobs:
     needs: [test]
     runs-on: ubuntu-latest
     steps:
-    - name: Download alpine node.js
+    - name: Download alpine node.js (x64 and arm64)
       uses: actions/download-artifact@v4
       with:
-        name: alpine_nodejs_${{github.event.inputs.NodeVersion}}
-    # Create GitHub release
-    - uses: actions/create-release@master
```

Comment:
This action is unmaintained and the repo has been archived: https://github.com/actions/create-release